content/Installation/C++.md

```diff
@@ -0,0 +1,90 @@
+---
+title: "C and C++"
+date: 2019-11-29T15:26:15Z
+draft: false
+weight: 20
+---
+
+You can install Enzyme using [Homebrew](https://brew.sh), or build it manually from source. To install using Homebrew, run
```

Comment:
Move both the homebrew and docker/dev container section to the main installation file (making this the "from source" file)